### PR TITLE
Block Explorer Support

### LINKED ISSUE #532: Unify and document status API interfaces

### PR DESCRIPTION
**Pull request by ejfitzgerald**
Add API to query TX contents

Also Closes #532

### CODE REVIEW

File: libs/ledger/src/tx_query_http_interface.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        }
+
+        // prepare the response
+        auto const &summary = tx.summary();
+
+        Variant tx_obj         = Variant::Object();
+        tx_obj["digest"]       = ToBase64(tx.digest());
+        tx_obj["fee"]          = summary.fee;
+        tx_obj["contractName"] = summary.contract_name;
+        tx_obj["data"]         = ToHex(tx.data());
```
Comment (pbukva) [inline libs/ledger/src/tx_query_http_interface.cpp]: Wouldn't it be better to use rather Base64 here as well?

Comment (ejfitzgerald) [reply libs/ledger/src/tx_query_http_interface.cpp]: Yeah I guess

**Review by pbukva — APPROVED** [review]
LGTM (I made one very minor comment, feel free to ignore)

**Comment by ejfitzgerald** [comment]
> LGTM (I made one very minor comment, feel free to ignore)

I have fixed

**Pull request merged**